pkg/fission-cli/main.go

```diff
@@ -122,13 +122,15 @@ func NewCliApp() *cli.App {
 	fnLogCountFlag := cli.StringFlag{Name: "recordcount", Usage: "the n most recent log records"}
 	fnForceFlag := cli.BoolFlag{Name: "force", Usage: "Force update a package even if it is used by one or more functions"}
 	fnExecutorTypeFlag := cli.StringFlag{Name: "executortype", Value: types.ExecutorTypePoolmgr, Usage: "Executor type for execution; one of 'poolmgr', 'newdeploy' defaults to 'poolmgr'"}
+	fnExecutionTimeoutFlag := cli.IntFlag{Name: "fntimeout, ft", Value: 60, Usage: "The length of time to wait for the response. If set to zero, by default it will wait of 60s for the response."}
```

Comment:
Usage -> 
```
The length of time to wait for the response. If the flag is not provided, by default it will wait of 60s for the response.
```

Reply:
Changed to this `Time duration to wait for the response while executing the function. If the flag is not provided, by default it will wait of 60s for the response.`